index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Portfolio</title>
+    <script src="https://kit.fontawesome.com/0ec53d8e70.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/dist/style.css">
+</head>
+<body>
+    <header class="navigation">
+      <div class="logo">
+        <h2>Evans Karanja</h2>
+      </div>
+      <nav class="navlinks">
+        <ul>
+          <li class="navItem" onclick="closeMenu()"><a href="#portfolio">Portfolio</a></li>
+          <li class="navItem" onclick="closeMenu()"><a href="#about">About</a></li>
+          <li class="navItem" onclick="closeMenu()"><a href="#contact">Contact</a></li>
+          <li class="navItem icn">
+            <a href="#email"><i class="fa-solid fa-envelope"></i></a>
+          </li>
+        </ul>
+      </nav>
+      <div class="toggle">
+        <div class="menu"></div>
+      </div>
+    </header>
+    <section class="intro">
+      <div class="intro-text">
+        <h1>Hey There.</h1>
+        <h1>I'm Evans</h1>
```

Comment:
[OPTIONAL] It is highly recommended to have only one `h1` tag on a whole which stands for the main title. I highly recommend you wrap these two tags into one tag which will increase your website accessibility. :wink: